src/libs/PolicyUtils.ts

```diff
@@ -71,9 +71,10 @@ Onyx.connect({
  * Filter out the active policies, which will exclude policies with pending deletion
  * These are policies that we can use to create reports with in NewDot.
  */
-function getActivePolicies(policies: OnyxCollection<Policy> | null): Policy[] {
+function getActivePolicies(policies: OnyxCollection<Policy> | null, currentUserLogin: string | undefined): Policy[] {
```

Comment:
Why? I have checked all instances where we have used `getActivePolicies` and additional check of roles is already the requirement. `shouldShowPolicy` is not the correct choice because it returns pending delete policies when offline. There is no way a policy where the current user has no role can be considered an active policy and updating getActivePolicies will avoid similar bugs in the future too.

Reply:
I don't see the advantage of doing that:
`getActivePolicies` accepts policies list and returns only policies that are active while `shouldShowPolicy` determines whether the policy should be shown or not (which allows pending delete policy when offline) and using additional param to use it for other purpose doesn't look like better approach to me a function should ideally serve one functionality.
@cristipaval WDYT about the above discussion

Reply:
I think it is important to maintain the single responsibility principle for the functions. So yes, I think your suggestion makes sense, @FitseTLT. On the other hand, I think the function name might be a bit confusing, and I can see @shubham1206agra's concern. Maybe we could update the function doc and mention that we also filter based on the role since the users can have in Onyx policies that they are not members of.